roles/convert2rhel/defaults/main.yml

```diff
@@ -18,3 +21,7 @@ foreman_convert2rhel_content_view: "Default Organization View"
 # Host groups
 foreman_convert2rhel_hostgroup7: "CentOS 7 converting"
 foreman_convert2rhel_hostgroup8: "CentOS 8 converting"
+foreman_convert2rhel_hg_oracle7: "Oracle 7 converting"
```

Comment:
I still think that the hostgroup variables should have identical structure. And my vote would be:
* foreman_convert2rhel_hostgroup_centos7
* foreman_convert2rhel_hostgroup_centos8
* foreman_convert2rhel_hostgroup_oracle7

And you're converting Oracle *Linux*, so while I think it's fine to shorten it in the variable name, the HG name should be "Oracle Linux 7 converting"